packages/driver/cypress/integration/e2e/multidomain_spec.ts

```diff
@@ -41,7 +40,148 @@ describe.skip('multidomain', () => {
     cy.switchToDomain('127.0.0.1:3501', () => {
       // @ts-ignore
       cy.now('get', '[data-cy="cypress-check"]').then(($el) => {
-        top.postMessage({ host: location.host, queriedText: $el.text() }, '*')
+        top.postMessage({ host: location.host, actual: $el.text() }, '*')
+      })
+    })
+  })
+
+  describe('window events', () => {
+    it('form:submitted', (done) => {
+      expectTextMessage('form:submitted', done)
+
+      // @ts-ignore
+      cy.switchToDomain('127.0.0.1:3501', () => {
```

Comment:
what's the reasoning behind not typing `cy.switchToDomain`?

Reply:
The implementation for it is basically a stub at this point. Its full and proper implementation will be done in a future effort, so I haven't bothered typing it. Its name is likely to change and its argument signature needs to be fleshed out and implemented (it's going to optionally take 2 more arguments), so the typing would just need to change in the future anyway.